flink-runtime/src/test/java/org/apache/flink/runtime/testutils/ZooKeeperTestUtils.java

```diff
@@ -68,8 +73,9 @@ public static Configuration configureZooKeeperHA(
 		config.setString(HighAvailabilityOptions.HA_ZOOKEEPER_QUORUM, zooKeeperQuorum);
 
 		int connTimeout = 5000;
-		if (System.getenv().containsKey("CI")) {
+		if (System.getenv().containsKey("CI") || System.getenv().containsKey("TF_BUILD")) {
```

Comment:
Do we have more checks like this? (if so, then maybe add a common utility for it?)

Reply:
A `git grep "\"CI\""` didn't reveal any other code locations.

Reply:
maybe introduce `runsOnCIInfrastructure()` to give this expression a more expressive name.